set.mm

```diff
@@ -650794,6 +650865,18 @@ D Fn ( I ... ( M - 1 ) ) /\
       ( wcel cop cxp opelxpi mp2an ) ACGBDGABHCDIGEFABCDJK $.
   $}
 
+  ${
+    $d ph f g x $.  $d R f g x $.  $d S f g x $.
+    ofeqd.1 $e |- ( ph -> R = S ) $.
+    $( Equality theorem for function operation, deduction form.  EDITORIAL:
+       Using this to prove ~ ofeq is shorter than the other way around.
+       (Contributed by SN, 11-Nov-2024.) $)
+    ofeqd $p |- ( ph -> oF R = oF S ) $=
```

Comment:
Then I'm in favor of moving it to Main and using it to prove ofeq and also minimize with it theorems using ofeq.